src/openrct2/config/Config.cpp

```diff
@@ -38,6 +38,12 @@
 using namespace OpenRCT2;
 using namespace OpenRCT2::Ui;
 
+#ifdef __APPLE__
```

Comment:
Not convinced people will like the button switching to the other side after 22 years of it being on the right.

Reply:
It's just the default that switches, and only for macOS users. I personally think it's a nice touch, and consistent with OS behaviour.

Reply:
I think we can default to left side for apple users and see what the reaction is like, I also have a feeling that they gonna appreciate this.

Reply:
I think OpenTTD always defaults to the left.